### PR TITLE
Add a file for Fairmont, MN

### PR DESCRIPTION
**Pull request by kristengemmill**
The purpose of this pull request is to add a hometown file for Fairmont, MN

Some :? that I want you all to help me answer:
1. If you have been to Fairmont, what did you do? What did you think of it?
1. What kind of food do you like?

### CODE REVIEW

**Comment by michaelchristianson-securian** [comment]
@kristengemmill 
I haven't been to Fairmont.  What part of MN is it in?
I like mexican food :burrito:

File: kristengemmill-fairmont.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Fairmont, MN
+
+### Resturants
+* Channel Inn
```
Comment (michaelchristianson-securian) [inline kristengemmill-fairmont.md:4]: :question: What type of restaurant is Channel Inn?

Comment (kristengemmill) [reply kristengemmill-fairmont.md:4]: Its a local bar/restaurant located right on one of the lakes in town

**Comment by kristengemmill** [comment]
@michaelchristianson-securian 
Fairmont is located in south central MN right on I-90.
If you like Mexican food, then you should go to El Agave Mexican if you ever visit

**Comment by kristengemmill** [comment]
This pull request resolves #18

**Review by michaelchristianson-securian — APPROVED** [review]
Looks good!

**Pull request merged**